tutorials/tutorial02_multiapps/doc/content/step01_multiapps.md

```diff
@@ -0,0 +1,305 @@
+a large number of positions a file can be provided instead using the
+`positions_file` parameter.
+
+
+- The $(x, y, z)$ coordinates are a vector that is being added to the coordinates of the sub-app's
+  domain to put that domain in a specific location within the master domain.
+- If the sub-app's domain starts at $(0,0,0)$ it is easy to think of moving that point around
+  using `positions`.
+- For sub-apps on completely different scales, `positions` is the point in the master domain where
+  that app is located.
+
+!---
+
+## MultiApp Position
+
+!media framework/multiapps_positions.png id=multiapps_pos style=width:80%;margin-left:auto;margin-right:auto;
+       caption=Example of MultiApp object position.
+
+!---
+
+## Parallel Execution
+
+The `MultiApp` system is designed for efficient parallel execution of hierarchical problems. The
+master application utilizes all processors.  Within each `MultiApp`, all of the processors are split
+among the sub-apps. If there are more sub-apps than processors, each processor will solve for
+multiple sub-apps.  All sub-apps of a given `MultiApp` are run simultaneously in parallel. Multiple
+`MultiApps` will be executed one after another.
+
+!---
+
+## Dynamically Loading Multiapps
+
+If building with dynamic libraries (the default) other applications can be loaded without adding them
+to your Makefile and registering them. Simply set the proper `type` in your input file
+(e.g. `AnimalApp`) and MOOSE will attempt to find the other library dynamically.
+
+- The path (relative preferred) can be set in your input file using the parameter
+  `library_path`; this path needs to point to the `lib` folder
+  within an application directory.
+- The `MOOSE_LIBRARY_PATH` may also be set to include paths for MOOSE to search.
+
+Each application must be compiled separately since the main application Makefile does not have
+knowledge of any sub-app application dependencies.
+
+!---
+
+## Basic MultiApp
+
+!row!
+!col! width=50%
+To get started, let's consider a simple system with two apps as shown on the right.
+
+For now: no Transfers, no Coupling
+
+Each application will march forward in time together, solve, and output
+!col-end!
+
+!col width=50%
+!media multiapps_01_hierarchy.png
+       style=width:70%;margin-left:auto;margin-right:auto;display:block;
+
+!row-end!
+
+!---
+
+## 01_master.i
+
+!listing step01_multiapps/01_master.i
+
+!---
+
+## 01_sub.i
+
+!listing step01_multiapps/01_sub.i
+
+Note how the `sub-app` input file doesn't even "know" it's being run within a MultiApp hierarchy!
+
+!---
+
+## Run 01_master.i
+
+- Look at the order of execution
+- Inspect outputs
+- Modify `execute_on` to see what happens
+
+!---
+
+## Sub-App Constraining dt
+
+By default the MultiApp system will "negotiate" a dt that makes sense for the entire hierarchy: it will choose the smallest dt that any app is currently requesting
```

Comment:
```suggestion
By default the MultiApp system will "negotiate" a timestep dt that makes sense for the entire hierarchy: it will choose the smallest dt that any app is currently requesting
```